resources/js/phonefield.js

```diff
@@ -1,4 +1,10 @@
 function phoneMask(fields) {
+    if (typeof (allowed_cities) != "undefined") {
+        var onlyCountriesOption = allowed_cities ? {onlyCountries: allowed_cities} : {};
+    }
+
+    var initialCountry = (typeof country_default !== "undefined") ? country_default : 'auto';
+
     var country = document.getElementById('default-phone-country')
```

Comment:
what do you think about defining onlyCountriesOption at the top of the block as {}, then set it inside same if block you checked on typeof?